### ISSUE TITLE
Remove built-in help docs, link to the docs.sourcegraph.com/@version instead

### ISSUE THREAD

**Issue by ryan-blunden** [sourcegraph/sourcegraph#2337]
There are several issues (#1885, #1720, #1912) caused by styling or functionality differences between the in-product help and docs.sourcegraph.com

I propose removing the in-product help and instead, link to the version of the docs the instance is running.

This has the following benefits:

 - Still provides version specific docs.
 - No styling and functionality consistency issues so we can close #1885, #1720 and #1912
 - No light and dark theme styles for help docs.
 - Removes [Sourcegraph's dependency on docsite](https://sourcegraph.com/search?q=repo:%5Egithub%5C.com/sourcegraph/sourcegraph%24+docsite+).

In summary removing the in-product help still provides users with version specific docs while saving effort now, and in the future.

**Comment by sqs** [comment]
Will also fix part of #1361

**Comment by slimsag** [comment]
Making this change would also allow us to undo recent pkg/markdown changes, which I have made a strong case for in https://github.com/sourcegraph/sourcegraph/issues/2369
